packages/card/src/component.tsx

```diff
@@ -20,7 +20,7 @@ export function Card(props: CardProps) {
       className: classNames(props.className, {
         [ccCard.card]: true,
         [ccCard.cardShadow]: !props.flat,
-        [ccCard.cardSelected]: props.selected,
+        [props.selected ? ccCard.cardSelected : ccCard.cardUnselected]: !props.flat,
```

Comment:
The flat variant would never get the selected state? 🤔

Reply:
But maybe it shouldn't, but did before? 😅

Reply:
It wouldn't? Then why 25 handles selected/unselected state for flat card? :D

Reply:
Before, a selected flat card would have got both `ccCard.selected` (row 23) and `ccCardFlatSelected` (row 25)? Right? 🤷😄